src/action/base/add_user_to_group.rs

```diff
@@ -22,6 +22,9 @@ pub struct AddUserToGroup {
 }
 
 impl AddUserToGroup {
+    pub fn typetag() -> &'static str {
+        "add-user-to_group"
```

Comment:
Is this supposed to  be a kebab-ified snake? :scream: 
```suggestion
        "add-user-to-group"
```
or
```suggestion
        "add_user_to_group"
```